.gitignore

```diff
@@ -10,3 +10,6 @@ coverage/
 ## Eth
 build/
 .0x-artifacts/
+
+## Test coverage
+coverage.json
```

Comment:
CA: 1, this is a leftover artifact from solidity-coverage I removed it when I pulled it out. https://github.com/ConsenSys/web3studio-bootleg/commit/524a5a0a32e8fbcffcea35dc89fbcce978f75055#diff-a084b794bc0759e7a6b77810e01874f2L9

The better option would be to just delete the file.

Reply:
Done!